service/header/subscription_test.go

```diff
@@ -36,9 +36,6 @@ func TestSubscriber(t *testing.T) {
 	err = p2pSub1.Start(context.Background())
 	require.NoError(t, err)
 
-	// subscribe
-	subscription, err := p2pSub1.Subscribe()
-	require.NoError(t, err)
 
```

Comment:
Remove one newline to please the linter (or run `gofmt -s` on the file):
```
service/header/subscription_test.go#L39
File is not `gofmt`-ed with `-s` (gofmt)
```

Reply:
No problem. Fixed.